src/main/java/io/vertx/mqtt/impl/MqttServerImpl.java

```diff
@@ -154,5 +154,6 @@ public void userEventTriggered(ChannelHandlerContext ctx, Object evt) throws Exc
         }
       }
     });
+    
```

Comment:
Can you please remove this line? Since it is not related to the PR

Reply:
ok